aws-lambda-layer/Rakefile

```
@@ -4,6 +4,7 @@ require 'fileutils'
 
 BUILD_DIR = File.join(__dir__, 'build')
 DEFAULT_FREETDS_VERSION = '1.1.6'
+DEFAULT_AWS_REGION = 'us-east-1'
```

What about something like

```suggestion
DEFAULT_AWS_REGION = ENV['AWS_REGION'] || 'us-east-1'
```

Also, is this compatible with a default region from .aws/credentials?

---

Yes, see "Setting the AWS Region" in https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html if you would like more information/confirmation.